packages/amplify_authenticator/test/ui/utils.dart

```diff
@@ -44,10 +44,44 @@ class AuthenticatorGoldenComparator extends LocalFileComparator {
     }
     if (!result.passed) {
       safePrint(
-        'A tolerable difference of ${result.diffPercent * 100}% was found when '
+        'D1 tolerable difference of ${result.diffPercent * 100}% was found when '
```

Comment:
What does D1 mean here?

Reply:
That was a cmd + f and replace mistake. Updated.